releasenotes/notes/profiling-add-pytorch-integration-0683123b7bb83f99.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features:
+  - |
+    profiling: Adds an experimental integration with the PyTorch profiler which can be enabled
+    by setting ``DD_PROFILING_PYTORCH_ENABLED=true``. This feature instruments the PyTorch 
+    profiler API (https://pytorch.org/docs/stable/_modules/torch/profiler/profiler.html)
+    so that GPU profiling data can be sent to Datadog for visualization.
+    This feature requires the environment variable ``DD_PROFILING_EXPORT_LIBDD_ENABLED=true``
```

Comment:
Should we say 'This feature also enables profiling native exporter' instead? Users don't need to set `DD_PROFILING_EXPORT_LIBD_ENABLED`. We will just auto enable it.

Reply:
sounds good, I removed mention of the native exporter here